### PR TITLE
[WIP] Refactor payment status mapping

### PR DESCRIPTION
**Pull request by motorina0**

### CODE REVIEW

**Comment by motorina0** [comment]
Note:

> I find PaymentStatusPending, PaymentStatusFailed and PaymentStatusSuccess better than PaymentPendingStatus, PaymentFailedStatus and PaymentSuccessStatus. Any thoughts?

File: tests/wallets/test_corelightningrest.py [hunk 1]
```diff
@@ -10,6 +11,13 @@
 ENDPONT = "http://127.0.0.1:8555"
 MCAROON = "eNcRyPtEdMaCaRoOn"
```
Comment (prusnak) [inline tests/wallets/test_corelightningrest.py]: Are these typos intentional?

Comment (motorina0) [reply tests/wallets/test_corelightningrest.py]: no :)
thanks!

File: tests/wallets/test_lndrest.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    + "djyukhys0z2rw6grgm8me9k4w9vn05zt"
+    + "9svzzjspp5ud2jdfpaqn5c2k2vphatsj"
+    + "ypfafyk8rcvkvwexnrhmwm94ex4jtqdq"
+    + "u24hxjapq23jhxapqf9h8vmmfvdjscqp"
+    + "jrzjqta942048v7qxh5x7pxwplhmtwfl"
+    + "0f25cq23jh87rhx7lgrwwvv86r90guqq"
+    + "nwgqqqqqqqqqqqqqqpsqyg9qxpqysgqy"
+    + "lngsyg960lltngzy90e8n22v4j2hvjs4"
+    + "l4ttuy79qqefjv8q87q9ft7uhwdjakvn"
+    + "sgk44qyhalv6ust54x98whl3q635hkwgsyw8xgqjl7jwu",
```
Comment (dni) [inline tests/wallets/test_lndrest.py:27]: yo actually don't need to `+`

**Comment by motorina0** [comment]
Dead meat!
PR will be re-created on top of the latest changes.
Closing.

**Pull request closed without merging**